Dockerfile

```diff
@@ -34,13 +34,15 @@ RUN set -ex \
 		libxmlsec1-dev \
 		locales \
 		locales-all \
+        media-types \
 		openssh-client \
 		p7zip-full \
 		rsync \
 		unar \
 		unzip \
 		xz-utils tk-dev \
 		zlib1g-dev \
+        rclone\
```

Comment:
This seems to be using spaces instead of tabs like the rest?

Reply:
Done. Oops my editor :(